far/findfile.cpp

```diff
@@ -515,9 +518,9 @@ void background_searcher::InitInFileSearch()
 	readBufferA.resize(readBufferSize);
 	readBuffer.resize(readBufferSize);
 
-	if (!Options.SearchHex)
+	if (!Params.DlgParams.Hex)
```

Comment:
There appears to be a typo here, causing this code to not be executed, which results in a never ending file search in `background_searcher::LookForString`. I tested this fix and the file search appears to be working correctly now.

```suggestion
	if (!Params.DlgParams.Hex.value())
```

Reply:
Ouch! Thank you! As I said, haven't had time to test it. Of course, you are right. That's going to be a bug farm; I should think of something less error-prone.

Reply:
I could not find a good solution which would not introduce lots of clutter in some other place(s). Leaving it as is for now (this place is fixed, of course).